.github/workflows/push-jar-maven.yml

```diff
@@ -0,0 +1,44 @@
+name: Upload Jar to Maven
+
+on:
+  push:
+    tags: 
+      - v*
+jobs:
+  upload-security-jar:
+    runs-on: [ubuntu-16.04]
+
+    name: Upload Jar to Maven
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        
+      - name: Configure AWS CLI
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+
+  # Since CD workflow uses this version of java
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11.0.x
+
+      - name: Uploading Jar to Maven
+        env:
+          passphrase: ${{ secrets.PASSPHRASE }}
+        run: |
+          cd ..
+          aws s3 cp s3://opendistro-docs/github-actions/pgp-public-key .
+          aws s3 cp s3://opendistro-docs/github-actions/pgp-private-key .
+          
+          gpg --import pgp-public-key
+          gpg --allow-secret-key-import --import pgp-private-key
+
+          aws s3 cp s3://opendistro-docs/github-actions/settings.xml /home/runner/.m2/
```

Comment:
Any reason why we want to copy the credentials from S3 instead of adding them to github secrets and reading from there ? 

https://help.github.com/en/actions/language-and-framework-guides/publishing-java-packages-with-maven

Reply:
Please check #481 (uses CD pipeline and sensitive data is added to github secrets)